### DISCUSSION TITLE
Question regarding programming the Streamdeck with the Flight Tracker

### DISCUSSION THREAD

**Discussion by Donkerg** [nguyenquyhy/Flight-Tracker-StreamDeck#314] [General]
Hi all. 

I,ve downloaded the Flight tracker plugin and also the Known- events and the Toggle-value s. 
But still I can`t get things working correctly.  I,m assuming that the buttons work for all planes ? 

As an example: 
For strobe I have: 
Title: Strb
Toggle Event : STROBES_TOGGLE
Feedback value: LIGHT_STROBE

This works fine. 

Now this one
Title: Taxi
Toggle Event:  TOGGLE_TAXI_LIGHTS
Feedback value : LIGHT_TAXI

This doesn`t seem to work, but it,s the same kind of switch as the Strobe, just on and of. 

Any ideas, what i,m doing wrong ? 

Greetings

**Comment by rmroc451** [comment]
which plane @Donkerg?  Some of the addon aircraft aren't utilizing the standard simconnect logic and implementing their own variables that we don't have access too without the MobiFlight WASM Module.

If you are using a plane shipped with MSFS, then it should work.  Attach a pic of the settings for the button, and we can help you out.

**Reply by Donkerg** [reply]
![taxibutton1](https://user-images.githubusercontent.com/36458895/143276190-da03e944-6f07-4441-b6a1-f7e861c879ee.png)

The plane is a standard 747.  ( KLM livery, but don`t think that that will matter ?

**Comment by rmroc451** [comment]
do the lights toggle in the sim when you press the button, but the button
on the deck doesn't display the value of the simulator appropriately?

**Reply by Donkerg** [reply]
There were spaces in the button configs..  my bad.  Copied the values to Excel and then there were spaces during the copy actins.

**Reply by rmroc451** [reply]
no worries!  I know there is an [item](https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/issues/308) in the backlog to address that. simple .trim 😁
